### PR TITLE
Polish magyar.lbx

### PR DESCRIPTION
**Pull request by moewew**
See our discussion in https://github.com/plk/biblatex/issues/717

### CODE REVIEW

**Comment by marczellm** [comment]
I cannot test this pull request. Use of the macro `\BiblatexHungarianWarningOff` in the lbx requires the latest version of biblatex.sty to be loaded, but I don't know how to run the build step that would write the date and version numbers into the sty file so that it does not error out with
```


! Missing number, treated as zero.
<to be read again>
D
l.7
A number should have been here; I inserted `0'.
(If you can't figure out why I needed to see a number,
look up `weird error' in the index to The TeXbook.)
! Missing = inserted for \ifnum.
<to be read again>
D
l.7
I was expecting to see `<', `=', or `>'. Didn't.
! Missing number, treated as zero.
<to be read again>
D
l.7
A number should have been here; I inserted `0'.
(If you can't figure out why I needed to see a number,
look up `weird error' in the index to The TeXbook.)
! LaTeX Error: Missing \begin{document}.
See the LaTeX manual or LaTeX Companion for explanation.
Type H <return> for immediate help.
...
l.7
You're in trouble here. Try typing <return> to proceed.
If that doesn't work, type X <return> to quit.
LaTeX Warning: You have requested, on input line 7, version
`' of package biblatex,
but only version
`DATE vVERSION programmable bibliographies (PK/MW)'
is available.
```

**Comment by moewew** [comment]
Just put
```latex
\makeatletter
\def\BiblatexHungarianWarningOff{\global\let\lbx@hu@loadinfo\@empty}
\makeatother
```
into your preamble.

**Comment by moewew** [comment]
Still, I have unpacked the definition of `\BiblatexHungarianWarningOff` in the warning handling to avoid problems if the `.lbx` file is dropped into a folder and used 'stand-alone'.

If you want to use the newest `.sty`, refer to https://github.com/plk/biblatex#installation

File: doc/latex/biblatex/biblatex.tex [hunk 1]
```diff
@@ -5119,7 +5120,7 @@ \subsubsection{Russian}
...
+The Hungarian localisation module needs to redefine certain field formats to obtain the grammatically correct word order. This means that these field formats are overwritten whenever the Hungarian localisation is active, no matter whether they were defined in the preamble or by a custom style. So please be aware that using the Hungarian localisation module may cause the bibliography output to deviate from the format dictated by the loaded style and preamble definitions. Changes to this behaviour need to be made using \cmd{DefineBibliographyExtras}. In particular \cmd{mkpageprefix} is redefined to output the page number as a suffix following Hungarian convention, additionally all formats of fields inolving pages were modified to make sure that page ranges are printed as ordinal ranges.
```
Comment (marczellm) [inline doc/latex/biblatex/biblatex.tex:5123]: "involving"
suffix?

(just marking this to fix later, I'll do it)

**Pull request merged**